apps/search-poc/src/components/TabList.tsx

```diff
@@ -4,21 +4,24 @@ import { Tabs } from 'antd';
 
 const { TabPane } = Tabs;
 
-type WorkspaceItem = {
+export type WorkspaceItem = {
```

Comment:
I suggest we move these type declarations to their own file. Looks like these are being used independent of the TabList.